docs/source/03_tutorial/04_create_pipelines.md

```diff
@@ -427,6 +477,7 @@ regressor:
 Versioning is enabled for `regressor`, which means that the pickled output of the `regressor` will be versioned and saved every time the pipeline is run. This allows us to keep the history of the models built using this pipeline. Further details can be found in the [Versioning](../05_data/02_kedro_io.md#versioning) section.
 
 ### Assemble the data science pipeline
+
 To create a modular pipeline for the price prediction model, add the following to the top of `src/kedro_tutorial/pipelines/data_science/pipeline.py`:
 
 ```python
```

Comment:
I know this isn't your change, but should't we just merge these two blocks so you can copy it all at once?

Reply:
Great point!